docs/Gemfile.lock

```diff
@@ -21,7 +21,7 @@ GEM
     execjs (2.7.0)
     faraday (0.14.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.21)
```

Comment:
What's this?

Reply:
https://github.com/improbable-research/keanu/network/alert/docs/Gemfile.lock/ffi/open